sos/policies/distros/__init__.py

```diff
@@ -997,10 +998,13 @@ def upload_s3(self, endpoint=None, region=None, bucket=None, prefix=None,
         if not secret_key:
             secret_key = self.get_upload_s3_secret_key()
 
+        boto3_config = BotocoreConfig(user_agent_extra='app/sos')
```

Comment:
I set it to `app/sos` because in a later version of boto3 (once Python 3.9 is the minimum supported version) the `user_agent_appid="sos"` would append "app/sos" to the user agent. This way in the future if or when sos migrates to using `user_agent_appid` the agent string for sos can remain consistent.